data/json/mapgen/house18.json

```diff
@@ -0,0 +1,152 @@
+            "add": [
+                { "item": "television", "x": 4, "y": 2, "chance": 55 },
+                { "item": "television", "x": 4, "y": 16, "chance": 20 }
+            ],
+            "comment": "randomly picked from a group ---v",
+            "place_items": [
+                { "item": "homebooks", "chance": 60, "x": [ 3,3 ], "y": [ 16,16 ]},
+                { "item": "homebooks", "chance": 60, "x": [ 5,5 ], "y": [ 16,16 ]},
+                { "item": "homebooks", "chance": 60, "x": [ 16,17 ], "y": [ 16,16 ]},
+                { "item": "magazines", "chance": 40, "x": [ 4,5 ], "y": [ 4,5 ]},
+                { "item": "magazines", "chance": 50, "x": [ 16,17 ], "y": [ 2,2 ]},
+                { "item": "novels", "chance": 50, "x": [ 10,10 ], "y": [ 5,5 ]},
+                { "item": "office", "chance": 70, "x": [ 17,19 ], "y": [ 22,22 ]},
+                { "item": "livingroom", "chance": 65, "x": [ 2,6 ], "y": [ 3,8 ]},
+                { "item": "dining", "chance": 70, "x": [ 17,19 ], "y": [ 5,6 ]},
+                { "item": "dining", "chance": 70, "x": [ 4,4 ], "y": [ 14,14 ]},
+                { "item": "dining", "chance": 70, "x": [ 19,19 ], "y": [ 14,14 ]},
+                { "item": "fridge", "chance": 60, "x": [ 5,5 ], "y": [ 14,14 ]},
+                { "item": "fridge", "chance": 60, "x": [ 18,18 ], "y": [ 10,10 ]},
+                { "item": "oven", "chance": 60, "x": [ 2,2 ], "y": [ 12,12 ]},
+                { "item": "oven", "chance": 60, "x": [ 20,20 ], "y": [ 12,12 ]},
+                { "item": "kitchen", "chance": 80, "x": [ 2,3 ], "y": [ 10,14 ]},
+                { "item": "kitchen", "chance": 80, "x": [ 19,20 ], "y": [ 10,14 ]},
+                { "item": "dresser", "chance": 70, "x": [ 2,2 ], "y": [ 18,19 ]},
+                { "item": "dresser", "chance": 70, "x": [ 2,2 ], "y": [ 22,22 ]},
+                { "item": "allclothes", "chance": 20, "x": [ 10,10 ], "y": [ 11,11 ]},
+                { "item": "allclothes", "chance": 20, "x": [ 12,12 ], "y": [ 11,11 ]},
+                { "item": "allclothes", "chance": 70, "x": [ 10,10 ], "y": [ 22,22 ]},
+                { "item": "allclothes", "chance": 70, "x": [ 12,12 ], "y": [ 22,22 ]},
+                { "item": "dresser", "chance": 70, "x": [ 20,20 ], "y": [ 17,17 ]},
+                { "item": "dresser", "chance": 70, "x": [ 20,20 ], "y": [ 20,20 ]},
+                { "item": "softdrugs", "chance": 60, "x": [ 8,10 ], "y": [ 19,20 ]},
+                { "item": "softdrugs", "chance": 60, "x": [ 12,14 ], "y": [ 19,20 ]},
+                { "item": "harddrugs", "chance": 10, "x": [ 8,10 ], "y": [ 19,20 ]},
+                { "item": "harddrugs", "chance": 10, "x": [ 12,14 ], "y": [ 19,20 ]},
+                { "item": "cleaning", "chance": 70, "x": [ 8,10 ], "y": [ 19,20 ]},
+                { "item": "cleaning", "chance": 70, "x": [ 12,14 ], "y": [ 19,20 ]},
+                { "item": "homeguns", "chance": 8, "x": [ 10,10 ], "y": [ 13,13 ]},
```

Comment:
`homeguns` will eventually be deprecated. Consider `guns_common` or possibly `guns_pistol_common` depending upon you preference. If you use `place_loot` then you can specify percentage change for `magazine` and `ammo`